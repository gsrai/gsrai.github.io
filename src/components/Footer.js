import React from 'react'

export default class Footer extends React.Component {
  render() {
    return (
      <div className='footer-container'>
        <hr />
        <footer className='footer'>&copy; 2017 Gagondeep Srai.</footer>
      </div>
    )
  }
}
