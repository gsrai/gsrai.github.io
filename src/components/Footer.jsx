import React from 'react';

export default class Footer extends React.Component {
  constructor(props) {
    super(props);
  }

  render() {
    return (
      <div className="footer-container">
        <hr />
        <footer className="footer">&copy; 2017 Gagondeep Srai.</footer>
      </div>
    );
  }

}