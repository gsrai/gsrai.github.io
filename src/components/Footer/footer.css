footer {
  text-align: center;
  margin-top: 1.5rem;
}

.footer-container {
  margin-top: 2rem;
}