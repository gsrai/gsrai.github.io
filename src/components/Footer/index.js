import React from 'react'

import './footer.css'

export default class Footer extends React.Component {
  render() {
    return (
      <div className='footer-container'>
        <hr />
        <footer className='footer'>&copy; 2018 Gagondeep Srai.</footer>
      </div>
    )
  }
}
