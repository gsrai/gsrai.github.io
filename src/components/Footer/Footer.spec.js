import { shallow } from 'enzyme'
import React from 'react'
import Footer from './index'

describe('Component Rendering', () => {
  test('component should render correctly', () => {
    const wrapper = shallow(<Footer />)
    expect(wrapper).toMatchSnapshot()
  })
})
