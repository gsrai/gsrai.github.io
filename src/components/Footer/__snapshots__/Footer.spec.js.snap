// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component Rendering component should render correctly 1`] = `
<div
  className="footer-container"
>
  <hr />
  <footer
    className="footer"
  >
    Â© 2018 Gagondeep Srai.
  </footer>
</div>
`;
